internal/pkg/cli/validate_test.go

```diff
@@ -191,6 +193,67 @@ func TestValidateDDBName(t *testing.T) {
 	}
 }
 
+func TestValidatePath(t *testing.T) {
+	mockFileSystem := func() {
+		fs := &afero.Afero{Fs: afero.NewOsFs()}
```

Comment:
This will create these files locally right? instead for testing we can use: `afero.NewMemMapFs()`